### DISCUSSION TITLE
feedback for weblate

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [jonjomckay/fritter#84] [General]
https://hosted.weblate.org/translate/fritter/fritter-android-metadata/fr/?checksum=737821d6d0f70fbe

other programmers at weblate have problems with such long strings.
And for the person who has to translate it, it is even more difficult than if you could still share it.
experiences of fdroid programmers and me.
just wanted to share this here because i had noticed or thought of it while translating.

**Comment by fitojb** [comment]
Yeah, if you contact Weblate dev Nijel he can help you sort out the default character limits :) @jonjomckay
